src/frontend/src/pages/stock/LocationDetail.tsx

```diff
@@ -139,10 +156,113 @@ export default function Stock() {
         label: t`Stock Locations`,
         icon: <IconSitemap />,
         content: <StockLocationTable parentId={id} />
+      },
+      {
+        name: 'default_parts',
```

Comment:
This should be hidden for the top-level category (i.e. no category specified)

Reply:
Good catch!